release/pkg/manager/calico/manager.go

```diff
@@ -708,7 +708,8 @@ func (r *CalicoManager) buildContainerImages(ver string) error {
 	return nil
 }
 
-func (r *CalicoManager) publishGithubRelease(ver string) error {
+func (r *CalicoManager) publishGithubRelease() error {
+	ver := r.calicoVersion
```

Comment:
We should just swap `ver` for `r.calicoVersion` instead of using a local variable.